### ISSUE TITLE
[RFC] Possible to add youtube parameters through oEmbed?

### ISSUE THREAD

**Issue by anketwokings** [bolt/bolt#6292]
Hi all, 

I use the video-field to embed a video. Since I do not want youtube to serve me related video's (rel=0) or the title (showinfo=0) I added the parameters to the video.
`https://www.youtube.com/watch?v=GgO2HkVGQmw&showinfo=0&rel=0`

This is, as @bobdenotter explained, ignored by embedly, that just serves a video with that ID.

On the embedly docs I found that it might support custom parameters: http://support.embed.ly/hc/en-us/articles/204266045-Custom-Video-Parameters

Could someone look into this and see if this is something that could be added to the bolt-embedly implementation? I think it would really be appreciated by a lot of people.

TLDR:
Working on a government site and I don't want to show the related video's after the video of the secretary of health has played. ;)

​​---------
 
- Relevant Bolt Version: [ 3.2 ]

**Comment by bobdenotter** [comment]
It's ugly as sin, but you can workaround this with the following: 

    {{ record.video.responsive|replace({'%3Ffeature': '%3Frel%3D0%26feature'})|raw }}

See also #3304.

**Comment by anketwokings** [comment]
That's a dirty hack but really helpful for now. ;)

**Comment by ToBe998** [comment]
It might be a good idea to change from embedly to the oembed API. This api is supported by almost all those services (youtube, twitter, vimeo, myvideo, and my others). Probably much better to use that and not a 3rd party like embedly, right?

**Comment by bobdenotter** [comment]
We've been wanting to do this for quite some time, but couldn't find the time to do so. If you can spare the time, this would be a welcome PR.

**Comment by ToBe998** [comment]
Problem is, we already did this, but as a different field type for our sir-trevor extension. I'll give it a look if it can be ported though.

**Comment by GawainLynch** [comment]
@bobdenotter what is missing now with the oEmbed API in play to bring this across the line?

**Comment by anketwokings** [comment]
I would still like to see this implemented, and still think this is a valid request.
Prefer to keep this one open! :)

Thanks

**Comment by GawainLynch** [comment]
Cool, by first the title should be renamed as since this was opened we replaced Embed.ly, and second it would be good to get an answer to my question above :wink:

**Comment by anketwokings** [comment]
Agreed! Title changed, and pinging @bobdenotter for the question:
> what is missing now with the oEmbed API in play to bring this across the line?

Thx!

**Comment by bobdenotter** [comment]
Let's keep this one open.
